### PR TITLE
Added a common var group and updated pipeline with new vars 

### PR DESCRIPTION
**Pull request by MerajNetCo**
Updated pipeline with new vars to disable eiir event-based triggers in staging slot

### CODE REVIEW

**Comment by jwyman85** [comment]
@MerajNetCo - Review comments as below

AzureWebJobs.ExtractJobNotificationTrigger needs to be:
Var name: AzureWebJobs.ExtractJobNotificationTrigger.Disabled
Var value: $(AzureWebJobsExtractJobNotificationTriggerDisabled)

AzureWebJobsRebuildIndexesDisabled
Has ‘.’’s in the var name in ADO var group so needs these removing to be consistent with the others
Var name: AzureWebJobs.RebuildIndexes.Disabled
Var value: $(AzureWebJobsRebuildIndexesDisabled)

I believe slot setting should be true rather than false (as we only want these configured on the deployment slot)

File: deploy-jobs-template.yml [hunk 1]
```diff
@@ -163,7 +163,32 @@ jobs:
...
                         "slotSetting": false
-                      }
+                      },
+                      {
+                        "name": "AzureWebJobs.ExtractJobScheduler.Disabled",
+                        "value": "$(AzureWebJobsExtractJobSchedulerDisabled)",
+                        "slotSetting": true
+                      },
+                      {
+                        "name": "AzureWebJobs.ExtractJobNotificationTrigger",
```
Comment (john5macd) [inline deploy-jobs-template.yml:173]: Missing .Disabled from end of this parameter.

I take this is a parameter from Library $(AzureWebJobsExtractJobSchedulerDisabled) - suggest this is not warranted, just set the values to true.  We never want to make these configurable via the library.

Comment (MerajNetCo) [reply deploy-jobs-template.yml]: Have corrected the var name. Library vars are not used just to configure the values, it's been done to keep the pipeline clean, tidy and lightweight.

**Review by jwyman85 — APPROVED** [review]
Approved

**Review by john5macd — APPROVED** [review]

**Pull request merged**